pkg/dumper/mysql/dumper.go

```diff
@@ -103,7 +103,12 @@ func (p *myDumper) insertIntoTable(txn *sql.Tx, tableName string, rowChan <-chan
 				if row[col] == nil {
 					rowValues[i] = "NULL"
 				} else {
-					rowValues[i] = string(row[col].([]uint8))
```

Comment:
any test for this?

Reply:
No. Testing only this branch is not feasible at this time. I did manual tests to make sure the referenced bug is fixed.

Reply:
@mereba You can add a test case in https://github.com/hellofresh/klepto/blob/master/features/mysql_test.go using a fixture that triggers the issue

Reply:
We can remove the chained if/else, also it's safer to have type assertion as follows:

```go
const (
  null = "NULL"
)

for i, col := range columns {
  switch row[col].(type) {
  case nil:
    rowValues[i] = null
  case string:
    rowValues[i] = row[col].(string)
  case []uint8:
    rowValues[i] = string(row[col].([]uint8))
  default:
    // log unknow type here
  }
}
```

Reply:
Thanks @rafaeljesus . [Done](https://github.com/hellofresh/klepto/pull/66/commits/4706bfc34b90ebbc27bb477c5cd791f4152e4556)